mofacts/client/views/home/profileDialogueToggles.js

```diff
@@ -19,6 +19,14 @@ Template.profileDialogueToggles.events({
     _state.set('selectedDialogueType',
         event.currentTarget.getAttribute('data-dialogue-type'));
   },
+
+  'click #confirmFeedbackSelection': function() {
+    Session.set('feedbackParamsSet',true);
+    Session.set('pausedLocks', 0);
```

Comment:
As we don't need to mess with pausedLocks, go ahead and delete this line too

Reply:
Still can reproduce the sound prompt playing exactly once after a paused lock is passed though, so at least we learned soemthing.